bittide/src/Bittide/Wishbone.hs

```diff
@@ -111,3 +117,165 @@ singleMasterInterconnect' config master slaves = (toMaster, bundle toSlaves)
     case divWithRemainder @(Regs a 8) @8 @7 of
       Dict ->
         f (master, unbundle slaves)
+
+unsafeToDf :: Circuit (CSignal dom (Maybe a)) (Df dom a)
```

Comment:
Everytime I read `unsafe` I hope that there's some comments either on the function or at the use-sites for 1) why it's safe to use and 2) why it's necessary

Reply:
Added a haddock comment :)